trade-finance-manager-ui/server/controllers/case/cancellation/helpers/get-previous-page-url-for-cancel-cancellation.ts

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Shall we change the file name to `get-previous-page-url` because we're already in the "case cancellation" directory", keep it simple?

Reply:
Great point, have changed